src/AElf.OS/Jobs/BlockSyncJob.cs

```diff
@@ -15,22 +17,27 @@ namespace AElf.OS.Jobs
 {
     public class BlockSyncJob
     {
-        private const long InitialSyncLimit = 10;
-        private const int BlockSyncJobLimit = 200;
+        private const int BlockSyncJobLimit = 10;
+        
+        private readonly TimeSpan _blockSyncJobAgeLimit = TimeSpan.FromSeconds(0.5);
 
         private readonly IBlockchainService _blockchainService;
         private readonly INetworkService _networkService;
         private readonly NetworkOptions _networkOptions;
         private readonly IBlockAttachService _blockAttachService;
         private readonly ITaskQueueManager _taskQueueManager;
+        private readonly INetworkSyncStateProvider _networkSyncStateProvider;
+        private readonly IAnnouncementCacheProvider _announcementCacheProvider;
 
         public ILogger<BlockSyncJob> Logger { get; set; }
 
         public BlockSyncJob(IBlockAttachService blockAttachService,
             IOptionsSnapshot<NetworkOptions> networkOptions,
             IBlockchainService blockchainService,
             INetworkService networkService,
-            ITaskQueueManager taskQueueManager)
+            ITaskQueueManager taskQueueManager,
```

Comment:
Too many injections in BlockSyncJob, maybe we should implement some logic in the service layer or domain layer.